test/models/CommonModel.spec.ts

```diff
@@ -479,6 +492,31 @@ describe('CommonModel', function() {
       });
     });
   });
+
+  describe('addToTypes', function() {
+    test('should add multiple types', function() {
```

Comment:
You need some many tests because you violated Single Responsibility Principle.

Reply:
Discussion in https://github.com/asyncapi/generator-model-sdk/pull/184#discussion_r632269074